src/Avalonia.Visuals/Media/Color.cs

```diff
@@ -14,6 +14,11 @@ namespace Avalonia.Media
 #endif
     readonly struct Color : IEquatable<Color>
```

Comment:
@jmacato Why is public conditional? Should HsvColor do the same? (I noticed this as I went to switch HsvColor to readonly which was missed at first).

Reply:
This type can be used in Avalonia.Build.Tasks project as included source file, but it won't be exported from this assembly.
So it's basically internal inside of Avalonia.Build.Tasks, but public inside of Avalonia.Visuals.

Reply:
Avalonia.Build.Tasks uses it for compile time parsing/

Reply:
uhm... im not really sure myself either haha. i think it's for CI/Release tests.

Reply:
Ok, well, I'm leaving HsvColor `public` until told otherwise. I know you were trying to combine a bunch of projects too to simplify things.

Reply:
I ended up copy/pasting what was done for Color. After referencing HsvColor from Color this was mandatory due to the way projects are setup.